tests/testthat/test-regression100Kcases.R

```diff
@@ -0,0 +1,125 @@
+context("Regression with 100,000 cases")
+
+vehicles <- readRDS("vehicles100kNoNA.rds")
+vehicles$region_weight <- vapply(as.character(vehicles$region),
+                                 function(x) switch(x,
+                                                    "Midwest" = 1.02,
+                                                    "Southeast" = 0.52,
+                                                    "West" = 1.09,
+                                                    "Southwest" = 1.68,
+                                                    "East" = 1.18),
+                                 0.0)
+set.seed(111)
+p.noise <- 15
+noise <- as.data.frame(replicate(p.noise, rnorm(nrow(vehicles))))
+colnames(noise) <- paste0("noise", 1:p.noise)
+vehicles <- cbind(vehicles, noise)
+
+test_that("Linear regression",
+{
+    expect_is(Regression(log.price~.-region_weight, data = vehicles),
+              "LinearRegression")
+    ## Effects Plot doesn't like "." in formula
+    expect_error(print(Regression(log.price ~ drive+condition+log.odometer+region+year,
+                   data = vehicles,
+                   weights = region_weight,
+                   output = "Effects Plot")), NA)
+    expect_error(print(Regression(log.price~.-region_weight,
+                   data = vehicles,
+                   weights = region_weight,
+                   output = "ANOVA")), NA)
+
+    expect_is(Regression(log.price~.-region_weight, data = vehicles,
+                         weights = region_weight),
+              "LinearRegression")
```

Comment:
Could perhaps make these tests faster by creating two regression objects and then referencing them in print statements instead of running the regression 4 times. The expanded formula could be used since the `.` is tested in `flipRegression`

Reply:
I also got this when running the file locally. Could perhaps update the tests to allow these warnings to appear.
![image](https://user-images.githubusercontent.com/4605715/117736584-4f689280-b23b-11eb-85f5-43123f1b978a.png)

Reply:
I have restructured this test_that block and added expectations for the warnings.